src/daria/utils/segmentation.py

```diff
@@ -40,30 +40,46 @@ def segment(
                 the image using rank based median, before the analysis.
             "rescaling factor" (float): factor how the image is scaled before
                 the actual watershed segmentation.
+            "monochromatic_color" (str): "gray" or "value", identifying the
+                monochromatic color space to be used in the analysis; default is gray.
 
     Returns:
         np.ndarray or daria.Image: labeled regions in the same format as img.
     """
 
     # ! ---- Preprocessing of input image
 
-    # Require scalar representation - work with grayscale image. Alternatives exist,
-    # but with little difference.
+    # Extract numpy array from image
     if isinstance(img, np.ndarray):
-        basis: np.ndarray = cv2.cvtColor(img, cv2.COLOR_RGB2GRAY)
+        basis: np.ndarray = np.copy(img)
     elif isinstance(img, daria.Image):
-        basis = cv2.cvtColor(img.img, cv2.COLOR_RGB2GRAY)
+        basis = np.copy(img.img)
     else:
         raise ValueError(f"img of type {type(img)} not supported.")
 
+    # Require scalar representation - the most natural general choice is either to
+    # use a grayscale representation or the value component of the HSV version,
+    # when.
+    monochromatic = kwargs.pop("monochromatic_color", "gray")
+    if monochromatic == "gray":
+        basis = cv2.cvtColor(basis, cv2.COLOR_RGB2GRAY)
+    elif monochromatic == "value":
+        hsv = cv2.cvtColor(img, cv2.COLOR_RGB2HSV)
```

Comment:
```suggestion
        hsv = cv2.cvtColor(basis, cv2.COLOR_RGB2HSV)
```

Reply:
I guess img still can be a darsia image?

Reply:
Yes.